roles/convert2rhel/tasks/content_views.yml

```diff
@@ -0,0 +1,104 @@
+    version: "1.0"
+    lifecycle_environments:
+      - "{{ foreman_convert2rhel_lifecycle_env }}"
+  when: foreman_content_rhel_enable_rhel7
+
+- name: "Create a '{{ foreman_convert2rhel_key_centos8 }}' content view"
+  theforeman.foreman.content_view:
+    username: "{{ foreman_username }}"
+    password: "{{ foreman_password }}"
+    server_url: "{{ foreman_server_url }}"
+    validate_certs: "{{ foreman_validate_certs | default(omit) }}"
+    organization: "{{ foreman_organization }}"
+    name: "{{ foreman_convert2rhel_key_centos8 }}"
+    repositories:
+      - name: '{{ foreman_convert2rhel_rhel8_repo }}'
+        product: '{{ foreman_convert2rhel_rhel8_product }}'
+  when: foreman_content_rhel_enable_rhel8
+
+- name: "Publish a '{{ foreman_convert2rhel_key_centos8 }}' content view"
+  theforeman.foreman.content_view_version:
+    username: "{{ foreman_username }}"
+    password: "{{ foreman_password }}"
+    server_url: "{{ foreman_server_url }}"
+    validate_certs: "{{ foreman_validate_certs | default(omit) }}"
+    organization: "{{ foreman_organization }}"
+    content_view: "{{ foreman_convert2rhel_key_centos8 }}"
+    version: "1.0"
+    lifecycle_environments:
+      - "{{ foreman_convert2rhel_lifecycle_env }}"
+  when: foreman_content_rhel_enable_rhel8
+
+- name: "Create a '{{ foreman_convert2rhel_key_oracle7 }}' content view"
+  theforeman.foreman.content_view:
+    username: "{{ foreman_username }}"
+    password: "{{ foreman_password }}"
+    server_url: "{{ foreman_server_url }}"
+    validate_certs: "{{ foreman_validate_certs | default(omit) }}"
+    organization: "{{ foreman_organization }}"
+    name: "{{ foreman_convert2rhel_key_oracle7 }}"
+    repositories:
+      - name: '{{ foreman_convert2rhel_oracle7_repo }}'
+        product: '{{ foreman_convert2rhel_oracle7_product }}'
+  when: foreman_convert2rhel_enable_oracle7
+
+- name: "Publish a '{{ foreman_convert2rhel_key_oracle7 }}' content view"
+  theforeman.foreman.content_view_version:
+    username: "{{ foreman_username }}"
+    password: "{{ foreman_password }}"
+    server_url: "{{ foreman_server_url }}"
+    validate_certs: "{{ foreman_validate_certs | default(omit) }}"
+    organization: "{{ foreman_organization }}"
+    content_view: "{{ foreman_convert2rhel_key_oracle7 }}"
+    version: "1.0"
+    lifecycle_environments:
+      - "{{ foreman_convert2rhel_lifecycle_env }}"
+  when: foreman_convert2rhel_enable_oracle7
+
+- name: "Create a '{{ foreman_convert2rhel_key_oracle8 }}' content view"
+  theforeman.foreman.content_view:
+    username: "{{ foreman_username }}"
+    password: "{{ foreman_password }}"
+    server_url: "{{ foreman_server_url }}"
+    validate_certs: "{{ foreman_validate_certs | default(omit) }}"
+    organization: "{{ foreman_organization }}"
+    name: "{{ foreman_convert2rhel_key_oracle8 }}"
+    repositories:
+      - name: '{{ foreman_convert2rhel_oracle8_repo }}'
+        product: '{{ foreman_convert2rhel_oracle8_product }}'
```

Comment:
don't you need both, `foreman_convert2rhel_rhel8_repo` AND `foreman_convert2rhel_oracle8_repo`, in the Oracle case?